OpenRA.Game/World.cs

```diff
@@ -615,8 +618,7 @@ public void Dispose()
 		public void OutOfSync()
 		{
 			EndGame();
-			SetPauseState(true);
-			PauseStateLocked = true;
+			ReplayTimestep = 0;
```

Comment:
Please add a comment like
```suggestion
			// Make sure the replay is completely frozen. Only set this here to not disable post-game chat in other situations.
			ReplayTimestep = 0;
```

Reply:
Added a comment